### ISSUE TITLE
Arm64: Investigate mapping FPCR and FPSR register 1:1 to the host

### ISSUE THREAD

**Issue by gdkchan** [Ryujinx/Ryujinx#4263]
Right now the results of floating-point operations are not fully correct in some cases because the rounding mode and other parameter does not match what the guest specifies. We should try to set the host control to the same value as the guest to ensure the floating-point operations will produce correct results. Ideally we should also pass the status values back to the guest.

We will likely need to leave some bits on those registers untouched, and we will likely need to save/restore the values on those registers when going in and out of emulator JIT generated code.

**Comment by ruje0504** [comment]
Yes, even if you manually install the firmware and configuration, you can't enter the game.

 Thanks you for you work.
